src/getAppSyncConfig.js

```diff
@@ -108,22 +107,16 @@ export default function getAppSyncConfig(context, appSyncConfig) {
 
         return {
           ...dataSource,
-          invoke: (payload) =>
-            invoke({
-              packageFolder: path.join(
-                context.serverless.config.servicePath,
-                context.options.location,
-              ),
-              handler: func.handler,
-              event: JSON.stringify(payload),
-              environment: {
-                ...(context.options.lambda.loadLocalEnv === true
-                  ? process.env
-                  : {}),
-                ...context.serverless.service.provider.environment,
```

Comment:
@seanvm Did you test environment variables??

env vars from `provider` should work (from serverless-offline) but it looks like we are losing support for `context.options.lambda.loadLocalEnv` here

See #22

cc/ @FilipPyrek

Reply:
@bboure I didn't see any issues with environment variables but support is already a bit limited with docker runtimes on serverless-offline.

I'll take a look at this with a node.js project to get a better idea of how it should be working.

Reply:
Hi guys, the goal of this feature `loadLocalEnv` is to propagate all available environment variables to the local execution of the function. It's mainly because of `AWS_*` environment variables for AWS SDK - read more in https://github.com/bboure/serverless-appsync-simulator/issues/22.

I did a tiny little test and it seems that serverless offline propagates all the environment variables, or doesn't it? If yes, I guess we can remove this configuration from `serverless-appsync-simulator`. 🤷‍♂️

Reply:
Thank you @seanmv and @FilipPyrek for your feedback!
I haven't had time to test this yet, unfortunately. I'll try to do this asap to make sure we didn't miss anything, but based on what you both just said it seems like this is a Go!